integration_test/scenarios/chat_detail_scenario.dart

```
@@ -0,0 +1,95 @@
+
+    for (final item in items) {
+      final row = item.root;
+      final owner = row.$(Text).at(1).text;
+      final displayName = row.$(Text).first.text ?? "";
+      final matrixAddress = row.$(Text).last.text ?? "";
+      const currentAccount  = String.fromEnvironment('CurrentAccount');
+      if(matrixAddress == currentAccount)
+      {
+        await verifyProfileInfoOfAMember(s, displayName, matrixAddress, isCurrentUser: true, level: parseUserLevel(owner));
+      }
+      else
+      {
+        await verifyProfileInfoOfAMember(s, displayName, matrixAddress, isCurrentUser: false, level: parseUserLevel(owner));
+      }
+      await ProfileInformationRobot($).backToGroupInfomationScreen();
+      await $.waitUntilVisible(item.root);
+    }
+  }
+
+  Future<void> verifyProfileInfoOfAMember(SoftAssertHelper s, String displayName, String matrixAdress, {bool isCurrentUser = true,
+    UserLevel level = UserLevel.member,}) async {
+    final memberRow = GroupInformationRobot($).getMember(matrixAdress);
+    await memberRow.tap();
+    await $.waitUntilVisible($(ProfileInfoView));
+    await verifyProfileInfo(s, displayName, matrixAdress, isCurrentUser: isCurrentUser, level: level);
+  }
+
+  Future<void> verifyProfileInfo(SoftAssertHelper s, String displayName, String matrixAdress, {bool isCurrentUser = true,
+    UserLevel level = UserLevel.member,}) async {
+    
+    s.softAssertEquals(ProfileInformationRobot($).getAvatar().exists, true, "Avatar is not shown");
+
+    // final expectedName = matrixAdress.substring(matrixAdress.indexOf("@")+1, matrixAdress.indexOf(":"));
+    final actualName = ProfileInformationRobot($).getDisplayName().text;
+    s.softAssertEquals( actualName == displayName
+    , true, "displayName is not correct, expected name is $displayName while actual name is $actualName",);
+        
+    try{
+      s.softAssertEquals(ProfileInformationRobot($).getOnlineStatus().exists, true, "online status is not shown");
+    }
+    catch(_){}
```

Empty catch blocks are allways fishy...